### PR TITLE
made the meter object deal with common time

### LINKED ISSUE #13: Common time isn't recognised:
The `m: C` (or `m: C|`) header tag described at http://abcnotation.com/wiki/abc:standard:v2.1#mmeter
is not supported, leaving the following error:

```python
Traceback (most recent call last):
  File "abctoguitar.py", line 26, in <module>
    tune = pyabc.Tune("X:0\n" + RAW_TUNE)
  File "/media/DATADIR/tims_code/pyabc/pyabc.py", line 500, in __init__
    self.parse_abc(abc)
  File "/media/DATADIR/tims_code/pyabc/pyabc.py", line 538, in parse_abc
    self.parse_tune(tune)
  File "/media/DATADIR/tims_code/pyabc/pyabc.py", line 563, in parse_tune
    self.tokens = self.tokenize(tune, self.header)
  File "/media/DATADIR/tims_code/pyabc/pyabc.py", line 579, in tokenize
    time_sig = TimeSignature(meter, unit, tempo)
  File "/media/DATADIR/tims_code/pyabc/pyabc.py", line 296, in __init__
    self._meter = [int(x) for x in meter.split('/')]
  File "/media/DATADIR/tims_code/pyabc/pyabc.py", line 296, in <listcomp>
    self._meter = [int(x) for x in meter.split('/')]
ValueError: invalid literal for int() with base 10: 'C'
```

### PR DESCRIPTION
**Pull request by wxtim**
Fix #13

### CODE REVIEW

File: pyabc.py [hunk 1]
```diff
@@ -293,6 +293,7 @@ def __sub__(self, x):
 
 class TimeSignature(object):
     def __init__(self, meter, unit_len, tempo=None):
+        meter = meter.replace('C|', '4/4').replace('C', '4/4')
```
Comment (wxtim) [inline pyabc.py]: This is the only real change. I seem to have been a little careless and one of my editors has must have some sort of auto linting, all the other changes are simply removing whitespace.

Comment (campagnola) [reply pyabc.py]: `C|` should be cut time `2/2`, correct?
http://abcnotation.com/wiki/abc:standard:v2.1#mmeter

Comment (wxtim) [reply pyabc.py]: I think you are, I'll correct this.

**Comment by wxtim** [comment]
@campagnola  I have done this.

**Pull request merged**